python/utils/bootstrap_utilities.py

```diff
@@ -223,9 +223,12 @@ def start_app(engine):
         app.setApplicationName("%s Python" % engine.context.project["name"])
 
         # set default icon
-        python_icon = os.path.realpath(os.path.join(
-            os.path.dirname(__file__),
-            "..", "..", "resources", "python_icon.png"))
+        python_icon = os.path.realpath(
```

Comment:
I'd suggest putting icons in the repo's root in an `icons` folder. Having them in `python/utils` doesn't feel right.